cli/connect/connect.go

```diff
@@ -33,6 +34,12 @@ func Enter(ctx *context.Ctx, args []string) error {
 
 	process := running.NewInstanceProcess(ctx, instanceName)
 	if !process.IsRunning() {
+		instanceNameParts := strings.SplitN(instanceName, ".", 2)
```

Comment:
Let's use the same approach as it was in util before: separate check for applicationName instead of instanceName and applicationName.instanceName instead of instanceName.

Reply:
You may extract it to a separate util like the one we used before, since it seems like we may reuse it in several places